Documentation/Configuration/Intercom/Whitelist.md

```diff
@@ -0,0 +1,51 @@
+    
+# Whitelist
+
+<kbd>  admintoolbox_intercom_whitelist  </kbd>  
+<kbd>  ServerRole:SpeakTime:CooldownTime  </kbd>
+
+<br>
+<br>
+
+## Description
+
+*Whitelist of server roles with speak limit and cooldown.*
+
+<br>
+
+### Roles
+
+The usable roles can be found in <br>
+the  `config_remoteadmin.txt`  file.
+
+<br>
+
+### Time
+
+Time is specified in seconds.
+
+<kbd>  1  </kbd>   **=̂**   <kbd>  1 Second  </kbd>
+
+<br>
+<br>
+
+## Example
+
+*A player with the  `TUTORIAL`  role will be able <br>
+to speak for  `4 Seconds` every  `300 Seconds`.*
+
+```yaml
+admintoolbox_intercom_whitelist : TUTORIAL:4:300
```

Comment:
This is only for Server Ranks & UserGroup, not ingame roles. As it is now at least...
Might add this though, its a good idea

Reply:
Where is the server rank listed btw?
https://github.com/ServerMod/Smod2/wiki/Enum-Lists

Reply:
These aren't listed, they are custom per server. It is the ranks used in `config_remoteadmin` in the main game.
https://scpsl-faq.com/fi/server-faq/faq#10-how-to-add-more-ranks-on-the-server